test/smoke_test/smoke_test.py

```diff
@@ -78,7 +79,7 @@ def smoke_test_cuda(package: str) -> None:
     if(package == 'all'):
         import torchaudio
         import torchvision
-        if installation_str.find("nightly") != -1:
+        if installation_str.find("nightly") != -1 or platform.system() == "Windows" :
```

Comment:
Please note I had to turn off this test for now on Windows since calling conda from python gives me an error with new CI worker.

Reply:
Thanks for the note.